tests/x11/yast2_lan_restart.pm

```diff
@@ -28,6 +28,8 @@ use y2lan_restart_common;
 use y2_module_basetest 'is_network_manager_default';
 use version_utils ':VERSION';
 
+my $backend = get_var('BACKEND');
```

Comment:
I suggest to either supply a default value or use `get_required_var`, e.g.:

```suggestion
my $backend = get_var('BACKEND', '');
```

or

```suggestion
my $backend = get_required_var('BACKEND');
```

http://open.qa/api/testapi/#_get_required_var

Reply:
Yes, I agree I will go with the `get_required_var`